doc/getting_started.rst

```diff
@@ -117,6 +117,12 @@ the test data::
   >>> accuracy_score(pipe.predict(X_test), y_test)
   0.97...
 
+.. note:: Diagram rendering of estimators
+    The default configuration for displaying a pipeline is 'text':
+    ``set_config(display='text')``. To visualize the diagram in Jupyter
+    Notebook, use ``set_config(display='diagram')`` and then call the pipeline
+    object.
```

Comment:
Unfortunately rendering the diagram in Getting Started is infeasible because it's not an image, it's interactive HTML. E.g. in the [user guide section 6.1](https://scikit-learn.org/dev/modules/compose.html#visualizing-composite-estimators), it doesn't show the interactive HTML as well.